AMP_CACHE_GUIDELINES.md

```diff
@@ -0,0 +1,85 @@
+# AMP Cache Guidelines
+
+## Purpose
+
+Provide guidelines to external parties for implementing their own AMP Caches.
+
+## Guidelines: Core
+
+1. An AMP Cache only serves [valid AMP documents](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md).
```

Comment:
AMP cache will only cache valid AMP docs. However, we actually serves invalid docs (small changes to serve versioned javascript).